lib/node_modules/@stdlib/stats/base/dists/negative-binomial/ctor/benchmark/benchmark.js

```diff
@@ -333,11 +394,15 @@ bench( pkg+':cdf', function benchmark( b ) {
 	r = 40;
 	p = 0.6;
 	dist = new NegativeBinomial( r, p );
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( 0.0, 1.0 );
+	}
 
 	b.tic();
 	for ( i = 0; i < b.iterations; i++ ) {
-		x = randu();
-		y = dist.cdf( x );
+		y = dist.cdf( x[i % len] );
```

Comment:
```suggestion
		y = dist.cdf( x[ i % len ] );
```

Make sure you have empty spaces at the beginning and end of brackets